src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx

```diff
@@ -0,0 +1,111 @@
+import type { FunctionComponent } from 'react';
+import React from 'react';
+import styled from 'styled-components';
+import map from 'lodash.map';
+
+import Input from 'components/Input';
+import Button from 'components/Button';
+
+import './style.scss';
+
+import Header from '../Header';
+import type { FormInputProps, FormMeta } from 'types/reactive-form';
+
+const allowedChars = /[\d,.;]+/;
+
+const filterInvalidKeys = (event: React.KeyboardEvent<HTMLInputElement>) => {
+  if (!allowedChars.test(event.key)) {
+    // Swallow invalid character inputs.
+    event.preventDefault();
+  }
+};
+
+interface ParentType { meta: { updateIncident: (data: any) => void }}
```

Comment:
This type is implemented in https://github.com/Amsterdam/signals-frontend/pull/1444, can you reuse that?